test/e2e/provisioning/all_e2e_test.go

```diff
@@ -205,7 +205,7 @@ func TestVsphereStaticIPProvisioningE2E(t *testing.T) {
 
 	// we only run one scenario, to prevent IP conflicts
 	scenario := scenario{
-		name:              "Coreos Docker Kubernetes v1.11.0",
+		name:              "Coreos v1.11.0",
```

Comment:
I renamed them because: We use the combination of `scenario.Name` and `cloudprovider` which is the string arg passed to the `testScenario` or `runScenarios` func as `metadata.Name` and as value for the `labelSelector` label. Labels however have a maximum lenght of 63 chars and because prow has longer build ids than circleCI this failed the tests for the `ubuntu upgrade` and `deployment-upgrade` testcases:

```
--- FAIL: TestUbuntuProvisioningWithUpgradeE2E (1.51s)
    helper.go:126: verify failed due to error=failed to verify creation of node for MachineDeployment: Internal error occurred: admission webhook "machinedeployments.machine-controller.kubermatic.io" denied the request: defaulting or validation failed: validation failed: [spec.selector.matchLabels: Invalid value: "ubuntu-docker-kubernetes-v1-10-5-ubuntu-upgrade-1063160795279396864": must be no more than 63 characters spec.selector: Invalid value: v1.LabelSelector{MatchLabels:map[string]string{"name":"ubuntu-docker-kubernetes-v1-10-5-ubuntu-upgrade-1063160795279396864"}, MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: invalid label selector.]
```